components/src/AppHeader/appheader.main.tsx

```diff
@@ -289,6 +313,18 @@ class AppHeaderMain extends Component<AppHeaderMainProps, AppHeaderMainState> {
             </Link>
           </div>
 
+          <div className="links-container">
```

Comment:
We may need to revisit this after getting more influence from @BonnieEP. There are some updates expected for the navigation header that we need to consider.